_inc/client/components/module-settings/index.jsx

```diff
@@ -68,69 +68,6 @@ export let SharedaddySettings = React.createClass( {
 
 SharedaddySettings = moduleSettingsForm( SharedaddySettings );
 
-export let RelatedPostsSettings = React.createClass( {
```

Comment:
@dereksmart I honestly don't think it's a regression. It's much better to tweak the options in the Customizer, seeing the related posts in context in the post and the site, being affected by any stylesheets the theme or other plugins might be loading than the sandboxed and inaccurate preview in the settings.
This is actually the whole point of this PR, if this doesn't sound good, let's close the PR.